asn.c

```diff
@@ -17,8 +18,20 @@
    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
+#include <gnutls/gnutls.h>
+#include <libtasn1.h>
+#include <stdlib.h>
+
+#include "asn.h"
+
 #include "rdesktop.h"
+// Generated by asn1Parser
+#include "pkix_asn1_tab.c"
```

Comment:
Sure.
I did not consider to make each commit distinct and buildable as the resulting commits are the product of merging and rebase anyway of commits in various privates branches.
So I think I can reorder and merge commits as you requested.